fixtures/server-config.postgres.json

```diff
@@ -0,0 +1,23 @@
+{
+	"server": {
+		"http_addr": ":4443",
+		"tls_key_file": "./notary-server.key",
+		"tls_cert_file": "./notary-server.crt"
+	},
+	"trust_service": {
+	  "type": "remote",
+	  "hostname": "notarysigner",
+	  "port": "7899",
+	  "tls_ca_file": "./root-ca.crt",
+	  "key_algorithm": "ecdsa",
+	  "tls_client_cert": "./notary-server.crt",
+	  "tls_client_key": "./notary-server.key"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "postgres",
+		"db_url": "postgres://server@serverdb:5432/notaryserver?sslmode=disable"
```

Comment:
Non-blocking:  if you know of a good/easy way to add SSL certs to the postgres server, that would be awesome, but from reading the docs it looks like we'd probably have to set the `PGDATA` directory to a local volume, and I'm not sure whether that's a good idea.